stellar-multitenant/internal/httphandler/tenants_handler_test.go

```diff
@@ -118,6 +166,13 @@ func Test_TenantHandler_Get(t *testing.T) {
 	tenant.DeleteAllTenantsFixture(t, ctx, dbConnectionPool)
 	tnt1 := tenant.CreateTenantFixture(t, ctx, dbConnectionPool, "myorg1", "GCTNUNQVX7BNIP5AUWW2R4YC7G6R3JGUDNMGT7H62BGBUY4A4V6ROAAH")
 	tnt2 := tenant.CreateTenantFixture(t, ctx, dbConnectionPool, "myorg2", "GB37V3J5C3RAJY6BI52MAAWF6AVKJH7J4L2DVBMOP7WQJHQPNIBR3FKH")
+	dTnt := tenant.CreateTenantFixture(t, ctx, dbConnectionPool, "dorg", "GBKXOCCQ5HXYOJ7NH5LXDKOBKU22TE6XOKHKYADZPRQFLR2F5KPFVILF")
```

Comment:
💡  Can you be more explicit in the variable name? `dTnt` will probably not make sense when someone look at this code months from now.